src/vcpkg/tools.cpp

```diff
@@ -306,10 +304,8 @@ namespace vcpkg
             cmd.string_arg(exe_path);
             return run_to_extract_version(Tools::NUGET, exe_path, std::move(cmd))
 #if !defined(_WIN32)
-                .map_error([](std::string&& error) {
-                    error.push_back('\n');
-                    error.append(msg::format(msgMonoInstructions).extract_data());
-                    return std::move(error);
+                .map_error([](LocalizedString&& error) {
+                    return std::move(error.append_raw('\n').append(msg::format(msgMonoInstructions)));
```

Comment:
I think this move needs to be rescoped:
```suggestion
                    return std::move(error).append_raw('\n').append(msgMonoInstructions);
```